test/Interop/Cxx/stdlib/fake-toolchain-module-interface.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-ide-test -print-module -module-to-print=FakeToolchain -tools-directory %S/Inputs/fake-toolchain/bin -source-filename=x -enable-experimental-cxx-interop -Xcc -stdlib=libc++ | %FileCheck %s
+// RUN: %target-swift-ide-test -print-module -module-to-print=FakeToolchain -tools-directory %S/Inputs/fake-toolchain/bin -source-filename=x -enable-experimental-cxx-interop -Xcc -stdlib=libc++ -Xcc -I%S/Inputs/fake-toolchain/include/c++/v1 | %FileCheck %s
```

Comment:
This doesn't look right – this test basically checks that passing libc++ include paths via `-I` or `-isystem` is not required when using `-stdlib=libc++`.

I wonder if this `-I` flag turns out to be needed on FreeBSD because of the lack of support for `-stdlib=libc++` in Clang Driver on this specific platform?

Either way, could you please disable the test on FreeBSD instead?